spec/models/post.spec.js

```diff
@@ -37,4 +37,19 @@ describe("Post model", () => {
       });
     });
   });
+
+  it("can upload and see the image", async (done) => {
+    let post = await new Post({ message: "image test", image_url: "https://picsum.photos/536/354" });
+
+    post.save((err) => {
+      expect(err).toBeNull();
+
+      Post.find((err, posts) => {
+        expect(err).toBeNull();
+
+        expect(posts[1]).toMatchObject({ message: "image test", image_url: "https://picsum.photos/536/354" });
```

Comment:
This test pre-supposes that there will be 2 posts in the database - suggest amending so that this finds the most recent post